cmd/utils/flags.go

```diff
@@ -881,11 +918,11 @@ func SplitAndTrim(input string) (ret []string) {
 // command line flags, returning empty if the HTTP endpoint is disabled.
 func setHTTP(ctx *cli.Context, cfg *node.Config) {
 	if ctx.GlobalBool(LegacyRPCEnabledFlag.Name) && cfg.HTTPHost == "" {
-		log.Warn("The flag --rpc is deprecated and will be removed in the future, please use --http")
+		log.Warn("The flag --rpc is deprecated and will be removed June 2021, please use --http")
```

Comment:
June 2021 is in the past :P

Is this already removed? 

Same for all subsquent flags here, they all mention same date

Reply:
go-ethereum prune this in the 1.10.9 (this PR, https://github.com/ethereum/go-ethereum/pull/23358, there is also another prunning in the 1.10.9).
I can swap that to "in the future" and wait for this in the merge of that version